### PR TITLE
feat(installations): show audit details

### PR DESCRIPTION
**Pull request by casey-brooks**
## Summary
- render installation status + audit log sections with safe markdown
- add audit log level formatting and pagination handling
- extend e2e mock server for AppsGateway audit APIs

## Testing
- npm run generate -- ../api --template buf.gen.yaml
- npm run lint
- npm run typecheck
- npm test
- npm run build
- VITE_OIDC_AUTHORITY=http://127.0.0.1:5001 VITE_OIDC_CLIENT_ID=console-app VITE_OIDC_SCOPE='openid profile email' npm run build
- npm run preview -- --port 4173
- MOCK_SERVER_PORT=5001 node test/e2e/mock-server.mjs
- E2E_BASE_URL=http://127.0.0.1:5001 METERING_GRPC_URL=http://127.0.0.1:50051 E2E_OIDC_AUTHORITY=http://127.0.0.1:5001 E2E_OIDC_CLIENT_ID=console-app E2E_OIDC_SCOPE='openid profile email' npm run test:e2e

## Issue
- #137

### CODE REVIEW

**Comment by casey-brooks** [comment]
## Test & Lint Summary
- `npm run generate -- ../api --template buf.gen.yaml`
- `npm run lint`
- `npm run typecheck`
- `npm test`
- `npm run build`
- `VITE_OIDC_AUTHORITY=http://127.0.0.1:5001 VITE_OIDC_CLIENT_ID=console-app VITE_OIDC_SCOPE='openid profile email' npm run build`
- `npm run preview -- --port 4173`
- `MOCK_SERVER_PORT=5001 node test/e2e/mock-server.mjs`
- `E2E_BASE_URL=http://127.0.0.1:5001 METERING_GRPC_URL=http://127.0.0.1:50051 E2E_OIDC_AUTHORITY=http://127.0.0.1:5001 E2E_OIDC_CLIENT_ID=console-app E2E_OIDC_SCOPE='openid profile email' npm run test:e2e`

Tests:
- unit: 50 passed, 0 failed, 0 skipped
- e2e: 31 passed, 0 failed, 0 skipped
Lint: no errors

**Comment by casey-brooks** [comment]
## Test & Lint Summary
- npm run lint
- npm run typecheck
- npm test
- npm run build
- VITE_OIDC_AUTHORITY='http://127.0.0.1:5001' VITE_OIDC_CLIENT_ID='console-app' VITE_OIDC_SCOPE='openid profile email' npm run build
- npm run preview -- --port 4173 --strictPort (background)
- MOCK_SERVER_PORT=5001 MOCK_PROXY_TARGET='http://127.0.0.1:4173' MOCK_METERING_PORT=50051 node test/e2e/mock-server.mjs (background)
- E2E_BASE_URL='http://127.0.0.1:5001' E2E_OIDC_AUTHORITY='http://127.0.0.1:5001' E2E_OIDC_CLIENT_ID='console-app' E2E_OIDC_SCOPE='openid profile email' METERING_GRPC_URL='http://127.0.0.1:50051' npm run test:e2e

Tests:
- unit: 50 passed, 0 failed, 0 skipped
- e2e: 31 passed, 0 failed, 0 skipped
Lint: no errors

File: src/components/UpdateInstallationDialog.tsx [hunk 1]
```diff
@@ -55,18 +62,42 @@ export function UpdateInstallationDialog({
...
       onOpenChange(false);
     },
     onError: (error) => {
       toast.error(error instanceof Error ? error.message : 'Failed to update installation.');
-    },
-  });
+		},
+	});
+
+	const auditLogQuery = useInfiniteQuery({
```
Comment (noa-lucent) [inline src/components/UpdateInstallationDialog.tsx]: [major] Test coverage: this adds new end-user surface area (status + audit log rendering + pagination) but there are no unit/component tests for the dialog. Please add a vitest + testing-library test that:
- renders the Status block only when installation.status is non-empty (after trimming)
- renders the Audit log table only when entries exist
- verifies pagination via nextPageToken (Load more fetches next page)
Mock appsClient.listInstallationAuditLogEntries to return deterministic pages.

File: src/components/UpdateInstallationDialog.tsx [hunk 2]
```diff
@@ -35,10 +40,12 @@ export function UpdateInstallationDialog({
   organizationId,
 }: UpdateInstallationDialogProps) {
   const queryClient = useQueryClient();
```
Comment (noa-lucent) [inline src/components/UpdateInstallationDialog.tsx:42]: [minor] Formatting: this file now mixes tabs and spaces heavily (e.g. state declarations and JSX indentation). Please run the repo formatter / keep indentation consistent to avoid long-term churn.

File: src/lib/format.ts [hunk 3]
```diff
@@ -107,9 +107,16 @@ export function summarizeContainers(containers: Array<{ status: ContainerStatus
 }
 
 export function formatAppVisibility(visibility: AppVisibility): string {
```
Comment (noa-lucent) [inline src/lib/format.ts:109]: [nit] Indentation in `formatAppVisibility` / `formatInstallationAuditLogLevel` switched to tabs while the rest of the file uses spaces. Please reformat for consistency.

File: test/e2e/mock-server.mjs [hunk 4]
```diff
@@ -1063,6 +1063,32 @@ function handleAppsGateway(method, _body, res) {
   if (method === 'ListInstallations') {
     return sendJson(res, 200, { installations: [], nextPageToken: '' });
   }
+  if (method === 'ListInstallationAuditLogEntries') {
```
Comment (noa-lucent) [inline test/e2e/mock-server.mjs:1116]: [minor] The mock server always returns an empty audit log/status, which makes it hard to validate the new UI in tests. If you add the recommended UI test (or future e2e coverage), consider returning at least one representative audit log entry + a non-empty status for a known installation id.

**Review by noa-lucent — CHANGES_REQUESTED** [review]
UI implementation looks aligned with the spec (status markdown + audit log table + load-more pagination), and `react-markdown` is a reasonable choice for safe rendering.

Requesting changes for one blocker:
- Add unit/component tests covering the new status + audit log rendering/pagination behavior (see inline comment).

Also left minor/nit notes about inconsistent indentation and improving the mock server for future coverage.

**Comment by casey-brooks** [comment]
## Test & Lint Summary
- npm run lint
- npm run typecheck
- npm test
- npm run build
- VITE_OIDC_AUTHORITY='http://127.0.0.1:5001' VITE_OIDC_CLIENT_ID='console-app' VITE_OIDC_SCOPE='openid profile email' npm run build
- npm run preview -- --port 4173 --strictPort (background)
- MOCK_SERVER_PORT=5001 MOCK_PROXY_TARGET='http://127.0.0.1:4173' MOCK_METERING_PORT=50051 node test/e2e/mock-server.mjs (background)
- E2E_BASE_URL='http://127.0.0.1:5001' E2E_OIDC_AUTHORITY='http://127.0.0.1:5001' E2E_OIDC_CLIENT_ID='console-app' E2E_OIDC_SCOPE='openid profile email' METERING_GRPC_URL='http://127.0.0.1:50051' npm run test:e2e

Tests:
- unit: 52 passed, 0 failed, 0 skipped
- e2e: 31 passed, 0 failed, 0 skipped
Lint: no errors

**Review by noa-lucent — APPROVED** [review]
Re-review: requested changes are addressed.

- Added unit/component tests covering status trimming + audit log rendering and pagination.
- Cleaned up formatting/indentation in the touched files.
- Mock server now returns representative installation/audit-log data for UI validation.

LGTM.
